apps/backend/db_patches/0166_AddProposalInternalComments.sql

```diff
@@ -0,0 +1,15 @@
+DO
+$$
+BEGIN
+	IF register_patch('0166_AddProposalInternalCommentTable.sql', 'Farai Mutambara', 'Adding proposal internal comment table', '2024-11-19') THEN
+		BEGIN
+			CREATE TABLE IF NOT EXISTS proposal_internal_comments (
+				comment_id SERIAL PRIMARY KEY,
+				proposal_pk INT REFERENCES proposals(proposal_pk) ON DELETE CASCADE,
```

Comment:
Does this need to be part of a composite key with the comment_id since we only expect one per proposal?